packages/@react-types/menu/src/index.d.ts

```diff
@@ -24,13 +23,10 @@ export interface MenuTriggerState {
 }
 
 export interface MenuTriggerProps {
-  ref?: RefObject<HTMLElement | null>,
-  type?: 'dialog' | 'menu' | 'listbox' | 'tree' | 'grid',
-  trigger?: 'press' | 'longPress',
-  align?: 'start' | 'end',
+  // trigger?: 'press' | 'longPress',
```

Comment:
remove commented-out code?

Reply:
Going to keep it to be added as future functionality.